### PR TITLE
fix: add simple example to Project Euler: Problem 2 challenge

### LINKED ISSUE #38753: Poorly worded challenge: Project Euler: Problem 2: Even Fibonacci Numbers
**Describe your problem and how to reproduce it:**
The current challenge description makes it sound like `n` terms have to be generated while what we're checking for is the current term to be less than `n`

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers

### PR DESCRIPTION
**Pull request by tosinibrahim96**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #38753

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md
Comment (Sky020) [inline curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md:18]:
```suggestion
For example, if <code>n = 8</code>, your function should sum the even-valued Fibonacci numbers between <code>1 and 8</code>, inclusive. So, <code>fiboEvenSum(8)</code> should return <code>10</code>.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Thanks, for getting this through, but I just suggest these small changes.

File: curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md
Comment (Sky020) [inline curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md:18]:
```suggestion
For example, if <code>n = 8</code>, your function should sum the even-valued Fibonacci numbers between <code>1</code> and <code>8</code>, inclusive. So, <code>fiboEvenSum(8)</code> should return <code>10</code>.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Sorry about this, but after viewing the lesson locally, this looked off. Also, it might need translation.

File: curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md
Comment (tosinibrahim96) [reply curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md:18]: no problem. I'll change that. I only speak English though. Will you open another issue for the translation then

Comment (Sky020) [reply curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md:18]: Let me clarify: What happens with challenges is anything that is not inside `<code></code>` tags, can be automatically translated (hopefully), or contributors in the other languages, know to manually translate elements not inside those tags.

Comment (tosinibrahim96) [reply curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md:18]: ok. I see. I'll use Google translate to change the text outside the `<code></code>` tag then. I'll update the PR when I'm done.

Comment (Sky020) [reply curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md:18]: No, you do not need to do that. The `code` tags just assist the other contributors. You do not need to translate anything for this PR.

Comment (tosinibrahim96) [reply curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md:18]: Ok. Thanks for clarifying. I'll push the correction to your suggested changes now

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍 . Please be aware: There is a merge freeze for almost all PRs. So, once that has been lifted, and this gets a second approval, then this PR can be merged.

Also, for future PRs, when someone makes a Review Request, often you can just click the _Commit Suggestion_ button attached to the message, and the commit will be done automatically. It is just easier.

**Comment by tosinibrahim96** [comment]
> 
> 
> LGTM 👍 . Please be aware: There is a merge freeze for almost all PRs. So, once that has been lifted, and this gets a second approval, then this PR can be merged.
> 
> Also, for future PRs, when someone makes a Review Request, often you can just click the _Commit Suggestion_ button attached to the message, and the commit will be done automatically. It is just easier.

Noted. Thanks so much for your help with this.  I really appreciate it

File: curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md
Comment (thecodingaviator) [inline curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md:18]: I'm not sure if we want it like this. Why not make the example a test?

**Comment by RandellDawson** [comment]
@tosinibrahim96 Thank you for the work you put into this PR.  Although there was some discussion on issue https://github.com/freeCodeCamp/freeCodeCamp/issues/38753 regarding changing the wording, it was later decided in the same issue to not add/change any wording of the challenge and instead create an additional test.  Unfortunately, no one noticed this PR was created before another PR (#39221) was approved and merged that just added the additional test discussed in the issue.

We still appreciate your efforts and hope to see future contributions from you.  For now, we are closing this PR.

Thank you for understanding.

**Pull request closed without merging**
